lib/rules/no-mixed-requires.js

```diff
@@ -0,0 +1,197 @@
+/**
+ * @author Raphael Pigulla
+ * See LICENSE file in root directory for full license.
+ */
+"use strict"
+
+// eslint-disable-next-line @mysticatea/node/no-unsupported-features/node-builtins
```

Comment:
This currently warns that this was added in Node v9.3.  However, [according to the docs](https://nodejs.org/api/modules.html#modules_module_builtinmodules), it looks like this was was backported to v8.10 and v6.13 as well. I verified that tests pass locally in Node v8.10 (the minimum engine this plugin supports).

'm not sure how often this changes, but I'm assuming it's best to query this from Node rather than keeping a  hardcoded list. What do you think?

Reply:
The supported Node.js versions of this plugin are [>=8.10.0](https://github.com/mysticatea/eslint-plugin-node/blob/b8f99452c0e1a808af7aa349d417dca0837e3a67/package.json#L6). Therefore, we cannot use it on some versions such as `9.0.0`-`9.3.0`. That's the reason for this warning.

Anyway, I think that it should be hardcoded because the lint results should not be different for each runtime Node.js version.

Reply:
Understood! Will update.